crates/nu-cli/src/completions/dotnu_completions.rs

```diff
@@ -33,48 +38,81 @@ impl Completer for DotNuCompletion {
         let start_with_backquote = prefix_str.starts_with('`');
         let end_with_backquote = prefix_str.ends_with('`');
         let prefix_str = prefix_str.replace('`', "");
+        // e.g. `./`, `..\`
+        let cwd_only = prefix_str
```

Comment:
Now that this is also being used for `/`, perhaps a name like `no_lib_dirs` would be better